.travis.yml

```diff
@@ -14,3 +11,10 @@ rvm:
 
 before_install:
   - gem install bundler
+
+script:
+  - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then
+      RUN_E2E_TESTS=true bundle exec rake
```

Comment:
@f2prateek For this to work, we'll have to remove the `RUN_E2E_TESTS` variable added to travis. 

Either that, or we set both `RUN_E2E_TESTS` **and** `RUNSCOPE_TOKEN` as encypted variables and remove this check.